azurerm/resource_arm_kubernetes_cluster.go

```diff
@@ -396,6 +396,17 @@ func resourceArmKubernetesCluster() *schema.Resource {
 							ForceNew:     true,
 							ValidateFunc: validate.CIDR,
 						},
+
+						"load_balancer_sku": {
```

Comment:
could we [add this new field to the documentation](https://github.com/terraform-providers/terraform-provider-azurerm/blame/master/website/docs/r/kubernetes_cluster.html.markdown#L213)? We'd tend to do this as:

```
* `load_balancer_sku` - (Optional) Specifies the SKU of the Load Balancer used for this Kubernetes Cluster. Possible values are `basic` and `standard`. Defaults to `standard`.
```

Reply:
✔️